contracts/child/MaticChildERC20.sol

```diff
@@ -0,0 +1,157 @@
+pragma solidity ^0.5.11;
+
+import { ChildToken, ERC20, LibTokenTransferOrder } from "./ChildChain.sol";
+
+
+contract ChildERC20 is ChildToken, ERC20, LibTokenTransferOrder {
+
+  event Deposit(
+    address indexed token,
+    address indexed from,
+    uint256 amount,
+    uint256 input1,
+    uint256 output1
+  );
+
+  event Withdraw(
+    address indexed token,
+    address indexed from,
+    uint256 amount,
+    uint256 input1,
+    uint256 output1
+  );
+
+  event LogTransfer(
+    address indexed token,
+    address indexed from,
+    address indexed to,
+    uint256 amount,
+    uint256 input1,
+    uint256 input2,
+    uint256 output1,
+    uint256 output2
+  );
+
+  // mainnet Matic Token          0x7d1afa7b718fb893db30a3abc0cfc608aacfebb0
+  address public constant TOKEN = 0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0;
+
+  uint256 public currentSupply = 0;
+  uint8 constant private _decimals = 18;
+
+  constructor() public {}
+
+  // can be called by anyone to set owner
+  function initConstructor ()
```

Comment:
Shouldn't be this only one time thing? People can call this even if we change ownership.

Reply:
we can add check with onlyOnce but calling it multiple time also not an issue. Any way I can add only once flag.

Reply:
added this flag